src/main/scala/is/hail/variant/Call.scala

```diff
@@ -39,6 +39,17 @@ object Call extends Serializable {
       call
     }
 
+  def genNonmissingValue: Gen[Call] =
+    for (v <- Variant.gen;
+      nAlleles = v.nAlleles;
+      nGenotypes = triangle(nAlleles);
+      c <- Gen.choose(0, nGenotypes - 1);
+      call = c.asInstanceOf[java.lang.Integer]
+    ) yield {
```

Comment:
Use `{` and `}` between the `for` and `yield` so that you can elide all the `;`s.

Reply:
I don't think `Call.genArb` should exist. @cseed?

`TCall`'s `genNonMissingValue` looks broken right now:
```scala
  override def genNonmissingValue: Gen[Annotation] = Call.genArb
```
`Call.genArb` can return a missing `Call`.

We should remove `genArb` and move this `genNonmissingValue` to `TCall` and make calls to `Call.genArb` call `TCall.genArb`